README.md

```diff
@@ -1,306 +1,61 @@
-## Usage
+For VirtualMachines, it also allocates a MAC Address for the primary interface if [masquerade](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=masquerade) or [slirp](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=slirp) binding mechanism is used.
 
-For test environment you can use the [development environment](#Develop).
+## The Why
 
-For Production deployment:
+Kubemacpool allows for collision-free Mac allocation to Pods and VirtualMachines, in a a Kubernetes cluster operating 
+with multiple CNI plugins.
 
-* Install any supported [Network Plumbing Working Group de-facto standard](https://github.com/k8snetworkplumbingwg/multi-net-spec) implementation, for example [Multus](https://github.com/intel/multus-cni).
-  To deploy multus on a kubernetes cluster with flannel cni:
-```bash
-kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/hack/multus/kubernetes-multus.yaml
-kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/hack/multus/multus.yaml
-```
+## Deployment and Usage
 
-* [CNI plugins](https://github.com/containernetworking/plugins) must be installed in the cluster, for example
-```bash
-kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/hack/cni-plugins/cni-plugins.yaml
-```
-
-* Download the project yaml and modify its mac range to avoid collisions with nearby clusters.
-  Finally, deploy the project by applying its yaml.
-```bash
-wget https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/config/release/kubemacpool.yaml
-mac_oui=02:`openssl rand -hex 1`:`openssl rand -hex 1`
-sed -i "s/02:00:00:00:00:00/$mac_oui:00:00:00/" kubemacpool.yaml
-sed -i "s/02:FF:FF:FF:FF:FF/$mac_oui:FF:FF:FF/" kubemacpool.yaml
-kubectl apply -f ./kubemacpool.yaml
-```
-
-### kubemacpool service
+You can choose to deploy this plugin on [local virtualized cluster](doc/deployment-on-local-cluster.md) or on your [arbitrary cluster](doc/deployment-on-arbitrary-cluster.md).
 
-Kubemacpool is set to allocate mac to [supported interfaces](#About) on Pods and VirtualMachines.
-The user can specify whether MAC address will be allocated to all VirtualMachines and Pods by default or not.
-This can be done by patching of the following snippets. Patched manifest will be on [testing manifests](config/test/kubemacpool.yaml):
+## Next Steps
 
-To set kubemacpool to allocate all VirtualMachines and Pods by default (see [opt-out Mode](#How-to-opt-out-kubemacpool-MAC-assignment-for-a-namespace-in-Opt-out-mode)):
-```bash
-cp config/default/mutatepods_opt_out_patch.yaml config/test/mutatepods_opt_mode_patch.yaml
-cp config/default/mutatevirtualmachines_opt_out_patch.yaml config/test/mutatevirtualmachines_opt_mode_patch.yaml
-make generate-test
-```
+Once Kubemacpool is deployed, you can allocate a mac by simply creating a Pod/VirtualMachine.
+For example, by creating a simple VirtualMachine:
```

Comment:
This sounds a bit repetitive, I would try to rephrase it

```suggestion
A simple VirtualMachine can be created by issuing the following commands:
```

Reply:
Done